.circleci/common.sh

```diff
@@ -49,12 +49,16 @@ function checkout_torch_pin_if_available() {
 
 function install_deps_pytorch_xla() {
   XLA_DIR=$1
+
   # Install ninja to speedup the build
   pip install ninja
 
   # Install libraries required for running some PyTorch test suites
   pip install hypothesis
   pip install cloud-tpu-client
+  pip install absl-py
+  pip install --upgrade numpy>=1.18.5
```

Comment:
Do you want to rebase and remove my `install_numpy` in this file?

Reply:
Great, will do!